pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -810,6 +818,13 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 					Template: filepath.Join(c.OVMFPath, EFIVarsSecureBoot),
 				}
 			} else {
+                                _, err1 := os.Stat(filepath.Join(c.OVMFPath, EFICode))
+                                _, err2 := os.Stat(filepath.Join(c.OVMFPath, EFIVars))
```

Comment:
Is `EFIVars` really necessary in the "insecure boot" case?

Reply:
I did not get the chance to find out the answer for the question, only do the check basing on the premise of the existing code. In the code, the `domain.Spec.OS.NVRam` uses that file path.